conference/migrations/0030_auto_20210507_2047.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 2.2.19 on 2021-05-07 18:47
```

Comment:
Please recreate the migration (version 0030 is already taken by now) and rename the file to better explain what the migration is about.

Reply:
renamed the migration to "0032_add_instructions_to_textcha_question_verbose_name.py"